ngraph/python/tests/test_onnx/test_backend.py

```diff
@@ -173,24 +170,18 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
      "OnnxBackendNodeModelTest.test_nonmaxsuppression_two_classes_cpu"),
     (xfail_issue_39662,
      "OnnxBackendNodeModelTest.test_nonmaxsuppression_two_classes_cpu",
-     "OnnxBackendNodeModelTest.test_slice_default_axes_cpu",
      "OnnxBackendNodeModelTest.test_scatter_elements_with_negative_indices_cpu",
-     "OnnxBackendNodeModelTest.test_constant_pad_cpu",
-     "OnnxBackendNodeModelTest.test_edge_pad_cpu",
-     "OnnxBackendNodeModelTest.test_reflect_pad_cpu",
-     "OnnxBackendNodeModelTest.test_top_k_cpu",
-     "OnnxBackendNodeModelTest.test_top_k_negative_axis_cpu",
-     "OnnxBackendNodeModelTest.test_top_k_smallest_cpu",
      "OnnxBackendNodeModelTest.test_constantofshape_int_shape_zero_cpu",
      "OnnxBackendNodeModelTest.test_gather_negative_indices_cpu"),
+    (xfail_issue_49753,
+     "OnnxBackendNodeModelTest.test_slice_default_axes_cpu"),
+    (xfail_issue_49754,
+     "OnnxBackendNodeModelTest.test_top_k_cpu",
+     "OnnxBackendNodeModelTest.test_top_k_negative_axis_cpu",
+     "OnnxBackendNodeModelTest.test_top_k_smallest_cpu"),
     (xfail_issue_33633,
      "OnnxBackendNodeModelTest.test_maxpool_2d_dilations_cpu"),
-    (xfail_issue_38086,
-     "OnnxBackendNodeModelTest.test_dynamicquantizelinear_min_adjusted_expanded_cpu",
-     "OnnxBackendNodeModelTest.test_dynamicquantizelinear_expanded_cpu",
-     "OnnxBackendNodeModelTest.test_dynamicquantizelinear_max_adjusted_expanded_cpu",
-     "OnnxBackendNodeModelTest.test_quantizelinear_cpu"),
-    (xfail_issue_34310,
+    (xfail_issue_49207,
```

Comment:
third appearance of: xfail_issue_49207

Reply:
3 time of using xfail_issue_49207 in separated blocks - I suppose that this is not intentional

Reply:
Thank you for pointing it out :)